spec/outputs/s3_spec.rb

```diff
@@ -204,4 +204,27 @@
       subject.multi_receive_encoded(events_and_encoded)
     end
   end
+
+  describe "aws service" do
+    context 'us-east-1' do
+      let(:region) { 'us-east-1' }
+      it "sets endpoint" do
+        expect( subject.send(:bucket_resource).client.config.endpoint.to_s ).to eql 'https://s3.us-east-1.amazonaws.com'
+      end
+    end
+
+    context 'us-east-1' do
```

Comment:
should it be `ap-east-1` instead of `us-east-1`?

Reply:
```suggest
context 'ap-east-1' do
```